apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -63,3 +80,35 @@ export const WarningLabel = ({children}) => (
 WarningLabel.propTypes = {
   children: React.PropTypes.node
 };
+
+/**
+ * Given a search query, generate a results list of animationProps objects that
+ * can be displayed and used to add an animation to the project.
+ * @param {string} searchQuery - text entered by the user to find an animation
+ * @return {Array.<SerializedAnimationProps>} - roughly
+ */
+function searchAnimations(searchQuery) {
+  // Make sure to generate the search regex in advance, only once.
+  // Search is case-insensitive
+  // Match any word boundary or underscore followed by the search query.
+  // Example: searchQuery "bar"
+  //   Will match: "barbell", "foo-bar", "foo_bar" or "foo bar"
+  //   Will not match: "foobar", "ubar"
```

Comment:
This will be interesting to test with the animation names we will use to see if we need a different type of search. It'll also be interesting to test if kids have ideas of what to search or if we'll need to add promps or suggestions. (I just read the spec said 1,000 - 10,000 images which is a ton)